app/user/profile/page.tsx

```diff
@@ -1,16 +1,21 @@
-import { Suspense } from "react"
+"use client"
 
 import DebuggingSession from "./DebuggingSession"
+import LoanList from "./LoanList"
 import LogoutButton from "./LogoutButton"
 
 const Page = () => {
   return (
-    <div className="content-container">
-      <h1 className="text-typo-heading-3 pb-5 font-bold">Profile</h1>
-      <Suspense fallback={<p>Loading...</p>}>
```

Comment:
I think the `< DebuggingSession> ` component requires the `<Suspense>` to be there. Unfortunately. But we just remove the suspense when we do not need the debugging anymore. It has to do with server/client relations and awaiting stuff and yada yada.
Maybe, if you would be so kind, could write a `@todo` about that we should remove the suspense when removing the debugging?